### DISCUSSION TITLE
Adding toolchain features using `with_cfg`

### DISCUSSION THREAD

**Discussion by plonter123** [fmeum/with_cfg.bzl#78] [General]
I am trying to add `features` using `with_cfg`, with no success.

This is what I tried (I'm not really trying to create `cc_static_library` rule, this is just an example):
```bazel
load("@with_cfg.bzl", "with_cfg")

_builder = with_cfg(native.cc_library)

# opt. 1: attribute `features`: built-in attributes cannot be overridden.
_builder.extend("features", ["fully_static_link"])

# opt. 2: Custom build settings can only be referenced via Labels, did you mean Label("//command_line_option:features")?
_builder.extend("//command_line_option:features", ["fully_static_link"])

cc_static_library, _cc_static_library = _builder.build()
```

Is there a way to add a feature using `with_cfg`?
I would appreciate your help.

**Comment by fmeum** [comment]
The first option should work, but it currently doesn't due to a bug. Could you verify that this is fixed by https://github.com/fmeum/with_cfg.bzl/pull/79?

**Reply by plonter123** [reply]
@fmeum Seems to work perfectly. Thanks for the quick response :)
Are you planning on releasing a tagged version soon?

**Reply by fmeum** [reply]
v0.2.4 is out now

**Reply by plonter123** [reply]
Great, thank you :grinning:
